modules/ROOT/pages/intro-platform-hosting.adoc

```diff
@@ -86,13 +87,21 @@ Not all Anypoint Platform components are supported in all cloud environments. Th
 
 MuleSoft provides three different options for hosting the Anypoint Platform runtime plane:
 
+* CloudHub 2.0
++
+CloudHub 2.0 is MuleSoft's cloud-based environment for hosting the Mule runtime and related services. CloudHub 2.0 enables you to deploy an API or a Mule application into a container-based platform that's managed by MuleSoft. You can deploy to MuleSoft's shared spaces or your own configurable private spaces for more control over what can interact with your applications.
++
+CloudHub 2.0 also provides high availability, ingress load balancers, AWS role-based access controls, and endpoint configuration. See xref:cloudhub-2::index.adoc[CloudHub 2.0] for more information.
++
+CloudHub 2.0 can only be managed only by the version of the cloud control plane that exists in the same environment (US cloud or EU cloud).
+
 * CloudHub
 +
-CloudHub is MuleSoft's cloud-based environment for hosting the Mule runtime server and related services. CloudHub enables you to deploy an API or a Mule application on a platform that’s managed by MuleSoft.
+CloudHub is MuleSoft's original cloud-based environment for hosting the Mule runtime server and related services. CloudHub enables you to deploy an API or a Mule application on a platform that’s managed by MuleSoft.
 +
 CloudHub also provides high availability, clustering and failover of your APIs and Mule applications and performs load balancing for them. See xref:runtime-manager::cloudhub.adoc[CloudHub] for more information.
 +
-CloudHub can only be managed only by version of the cloud control plane that exists in the same environment (US cloud, EU cloud, or MuleSoft Government Cloud).
+CloudHub can only be managed only by the version of the cloud control plane that exists in the same environment (US cloud, EU cloud, or MuleSoft Government Cloud).
```

Comment:
this does not read grammatically pleasing. There are two "only's in one sentence.

Reply:
```suggestion
CloudHub can be managed only by the version of the cloud control plane that exists in the same environment (US cloud, EU cloud, or MuleSoft Government Cloud).
```